tests/v4l2decode.cpp

```diff
@@ -519,9 +521,11 @@ bool takeOneOutputFrame(int fd, int index = -1/* if index is not -1, simple enqu
 #elif __ENABLE_WAYLAND__
         ret = displayOneVideoFrameWayland(fd, &buf);
 #else
-        if (IS_DMA_BUF() || IS_DRM_NAME())
+        if (IS_DMA_BUF() || IS_DRM_NAME()) {
+#ifdef __ENABLE_TESTS_GLES__
             ret = displayOneVideoFrameEGL(fd, buf.index);
-        else
+#endif
+        } else
```

Comment:
when __ENABLE_TESTS_GLES__ is not defined the whole if{} else will be a no operation.  The #ifdef has to be located outside the if (IS_DMA_BUF()...)